.github/workflows/tests.yml

```diff
@@ -36,21 +36,10 @@ jobs:
           # Select PHPUnit version suitable for PHP 7.2.
           tools: composer, phpunit:^8.5
 
-      # Cache Composer Dependencies
-      # https://github.com/marketplace/actions/setup-php-action#cache-composer-dependencies
-      - name: Get composer cache directory
-        id: composer-cache-dir
-        run: echo "::set-output name=dir::$(composer config cache-dir)"
-      - name: Apply dependency caching
-        id: composer-cache
-        uses: actions/cache@v2
+      - uses: "ramsey/composer-install@v1"
```

Comment:
takes care of the caching logic

Reply:
https://github.com/ramsey/composer-install looks good to me, thank you.